app/models/reservation.rb

```diff
@@ -18,7 +18,8 @@ class Reservation < ActiveRecord::Base
   validate :matched_item_and_model
   validate :check_status
   validate :status_final_state
-  validate :not_in_past, :available, :check_banned, on: :create
+  validate :not_in_past, :available, :check_banned,
+           :check_consecutive, on: :create
```

Comment:
this might be overkill but we could add a `forced_update` flag for manual edits? And we actually probably don't need to check requests, because they require human checking anyway so the problem that this is attempting to solve won't arise. We could also not check it if the status is changing?